index.d.ts

```diff
@@ -122,8 +122,12 @@ export interface CancelTokenSource {
   cancel: Canceler;
 }
 
+export interface AxiosInterceptorOptions {
+  synchronous: boolean;
+  runWhen: (config: AxiosRequestConfig) => boolean;
```

Comment:
There 2 options are also optional, which means users can only set `synchronous` or `runWhen`.

Reply:
@sirctseb can you have a look at this?